2022q1/Makefile

```diff
@@ -0,0 +1,43 @@
+# Report month start
+START?=		1
+# Report month end
+STOP?=		3
+# Report year
+YEAR?=		2022
+# Report quarter
+QUARTER?=	1
+
+# Entries from the various official and semi-official teams
+TeamReports= FreeBSD-Foundation.adoc releng.adoc clusteradm.adoc ci.adoc portmgr.adoc
+
+# Projects that span multiple categories, from the kernel, and userspace to
+# the Ports Collection or external projects
+Projects= accessibility.adoc
+
+# Changes affecting the base system and programs in it
+Userland= boot-performance.adoc
+
+# Updates to kernel subsystems/features, driver support, filesystems,
+# and more
+Kernel= ena.adoc gunion.adoc rtw88.adoc iwlwifi.adoc ocf-wg.adoc
+
+# Updating platform-specific features and bringing in support for new
+# hardware platforms
+Architectures= dpaa2.adoc
+
+# Noteworthy changes in the documentation tree, man-pages, or new
+# external books/documents
+Documentation=	doceng.adoc
+
+# Changes affecting the Ports Collection, whether sweeping changes that
+# touch most of the tree, or individual ports themselves
+Ports= kde.adoc office.adoc gcc.adoc portconfig.adoc fpart.adoc wifibox.adoc
+
+# Third party project built on FreeBSD, but not part of the FreeBSD
+# project itself
+ThirdParty=
+
+# Objects that defy categorization
+Miscellaneous= hellosystem.adoc pot.adoc
+
+.include <../tools/Makefile>
```

Comment:
That file doesn't exist. (Or doesn't exist yet, at least.)

But more importantly, all the documents are listed here, except intro.adoc, which may have been left out deliberately.

Reply:
Thanks for reviewing pauamma.

Indeed, tools/Makefile does not exist yet: I am still working on it. As we are already late for 2022q1 and I need revision for the reports sorting, I thought it might be a good idea to send this file in advance for review, while I am finishing the rest.

As you guessed, intro.adoc is special and will be treated differently: it does not belong to any category and is always the first file to be included, there is no need for a variable in this case. So its absence is intended.